index.html

```diff
@@ -0,0 +1,571 @@
+
+As with [semver](https://semver.org/spec/v2.0.0.html), the `0.x` version series is special: there is no expectation of compatibility between versions `0.x` and `0.y`. For example, a processor must not activate implementation `0.4` to satisfy a requested version of `0.2`.
+
+### Ordering
+
+Versions can be ordered with the following pseudocode:
+
+```typescript
+compareVersions(a: Version, b: Version) -> LESS_THAN | EQUAL | GREATER_THAN {
+  if a.major < b.major {
+    return LESS_THAN
+  }
+  if a.major == b.major {
+    if a.minor == b.minor { return EQUAL }
+    if a.minor > b.minor { return GREATER_THAN }
+    return LESS_THAN
+  }
+  if a.major > b.major {
+    return GREATER_THAN
+  }
+}
+```
+
+Note that this ordering does *not* guarantee that if A > B then A can satisfy a version requirement of B. For example, `2.0 > 1.9`, but `2.0` cannot [satisfy](#satisfying-version-requirements) the requirements of a document which references version `1.9`, since the major version update introduced possibly-breaking changes.
+
+### Satisfaction
+
+Given a version `requested` by a document and an `available` version of an implementation, the following pseudocode will determine if the `available` version can satisfy the `requested` version:
+
+```typescript
+canSatisfy(requested: Version, available: Version) =
+  requested.major == available.major &&
+  (requested.major > 0 && (requested.minor <= available.minor)
+    || (requested.minor == available.minor))
+```
+
+### Referencing versions and activating implementations
+
+Schema documents **must** reference a feature version which supports all the schema elements and behaviors required by the document. As a practical matter, authors should generally prefer to reference the [lowest](#ordering) versions which can [satisfy](#satisfaction) their requirements, as this imposes the fewest constraints on processors and is thus more likely to be supported. Authors **may** choose a higher version which they believe has sufficient support.
+
+If a processor chooses to activate support for a feature, the processor **must** activate an implementation which can [satisfy](#satisfaction) the version required by the document.
+
+### Backus-Naur Form for versions and version tags
+
+```plaintext
+#::[BNF grammar for SemVer specifiers]
+‹version tag› ::= "v" ‹version›
+
+‹version› ::= ‹major› "." ‹minor›
+
+‹major› ::= ‹numeric identifier›
+‹minor› ::= ‹numeric identifier›
+‹numeric identifier› ::= "0"
+                       | ‹positive digit›
+                       | ‹positive digit› ‹digits›
+‹digits› ::= ‹digit›
+           | ‹digit› ‹digits›
+
+‹digit› ::= "0"
+          | ‹positive digit›
+‹positive digit› ::= "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
+```
+
+## Extensibility
```

Comment:
This whole section would be more clear with a realistic example of why you would do these things.